Source/Core/Core/PowerPC/JitArm64/Jit.h

```diff
@@ -182,6 +182,8 @@ class JitArm64 : public JitBase, public Arm64Gen::ARM64CodeBlock, public CommonA
 
   bool IsFPRStoreSafe(size_t guest_reg) const;
 
+  void rlwinmx_internal(UGeckoInstruction inst, u32 sh);
```

Comment:
Is this intended to be public?

Reply:
Well, I guess it has no real reason to be public... All these other helper methods already were, though. What do you think, should I move this one to private?

Reply:
tbh, the other respective helpers should also reasonably be private if they have no need to be public. I'm fine merging as is, if you'd prefer to do that as it's own thing

Reply:
Yeah, I think I'll do that in a separate PR then.